### PR TITLE
Contact us

### LINKED ISSUE #76: Contact us page
Enter a message that is sent directly to an e-mail address to be supplied by @Vince-Egan

### PR DESCRIPTION
**Pull request by sonlinux**
fix #76 

Add contact us feature

contact_us

### CODE REVIEW

File: django_project/base/templates/landing_page.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
 {% block body_content %}
 <div id="section-wrapper">
     <header class="masthead text-center text-white d-flex" id="header">
-        {% include "carousel.html" %}
+{#        {% include "carousel.html" %}#}
```
Comment (dimasciput) [inline django_project/base/templates/landing_page.html]: Why did you comment this?

File: django_project/core/settings/contrib.py [hunk 2]
```diff
@@ -87,3 +87,6 @@
 CELERY_RESULT_SERIALIZER = 'json'
 
 BROKER_URL = 'amqp://guest:guest@%s:5672//' % os.environ['RABBITMQ_HOST']
+
+# Contact us settings
+CONTACT_US_EMAIL = 'alison@kartoza.com'
```
Comment (dimasciput) [inline django_project/core/settings/contrib.py]: Please don't put your email here

File: django_project/base/templates/account/signup.html [hunk 3]
```diff
@@ -1,33 +1,12 @@
 {% load staticfiles %}
 {% load i18n %}
-{% load account socialaccount %}
```
Comment (dimasciput) [inline django_project/base/templates/account/signup.html]: Why did you remove these lines?

File: django_project/core/urls.py [hunk 4]
```diff
@@ -23,6 +23,7 @@
     url(r'^accounts/', include('allauth.urls')),
     url(r'^', include('bims.urls')),
     url(r'^api-auth/', include('rest_framework.urls')),
+    url('^contact/', include('contactus.urls')),
```
Comment (dimasciput) [inline django_project/core/urls.py]: Where did this come from?

Comment (sonlinux) [reply django_project/core/urls.py]: Okay @dimasciput am fixing these and the urls are coming from the third party app for contact us.
Thanks

File: django_project/base/templates/landing_page.html (hunk 1 above)
Comment (sonlinux) [reply django_project/base/templates/landing_page.html]: I have uncommented it, it was because I was using an outdated django-bims install which did not hav e carousel yet. After updating the package its fine.

File: django_project/core/settings/contrib.py (hunk 2 above)
Comment (sonlinux) [reply django_project/core/settings/contrib.py]: Alright, ill keep that in mind, 
Thanks

**Comment by timlinux** [comment]
Status update please @sonlinux I'm looking for four or five of these PR's a day so please keep the turn around time tight. The background of the form should be darker / semi opaque so that the text is clear to read for the labels.

File: django_project/base/templates/landing_page.html (hunk 1 above)
Comment (timlinux) [reply django_project/base/templates/landing_page.html]: Its still there if it shows here @sonlinux

**Comment by timlinux** [comment]
@sonlinux please look at @ann26 's sign up page here https://github.com/kartoza/LEDET_BIMS/pull/87 and make your UI consistent with hers.

**Comment by timlinux** [comment]
UI looks much better now thanks @sonlinux - you still have some unresolved comments from @dimasciput

**Comment by sonlinux** [comment]
Thanks @timlinux 
I have resolved them all now.

File: django_project/base/templates/account/signup.html (hunk 3 above)
Comment (ann26) [reply django_project/base/templates/account/signup.html]: I think this lines is needed, please bring it back

**Comment by timlinux** [comment]
Please move to kartoza-bims - ledet-bims should only be the LEDET specific skinning and apps which are not generic to the platform.
